### PR TITLE
Added Company model

### PR DESCRIPTION
**Pull request by make-it-git**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #53

Notify the following people: @darrynten

### CODE REVIEW

**Comment by make-it-git** [comment]
There is invalid warning from codeclimate

**Review by darrynten — CHANGES_REQUESTED** [review]
Looking good @make-it-git it just needs the README to be updated with the models checked off

**Comment by make-it-git** [comment]
@darrynten updated

File: src/Models/Company.php [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+     * @return Company
+     */
+    public function current()
+    {
+        $result = $this->request->request('GET', $this->endpoint, 'Current');
+        $model = new Company($this->config);
+        $model->loadResult($result);
+        return $model;
+    }
+}
```
Comment (darrynten) [inline src/Models/Company.php:574]: What are we going to do about the `GET Company/Get?includeStatus={includeStatus}` call?

**Review by darrynten — CHANGES_REQUESTED** [review]

File: src/Models/Company.php (hunk 1 above)
Comment (make-it-git) [reply src/Models/Company.php:574]: $model->all(['includeStatus' => true])

Comment (make-it-git) [reply src/Models/Company.php:574]: @darrynten check latest commit https://github.com/darrynten/sage-one-php/pull/66/commits/536eaf1be8ef20a9297f0e2012152d0cb08fd9ec

Comment (darrynten) [reply src/Models/Company.php:574]: OK great! Can you add some information about this to the README as it isn't immediately clear, but that's very nice, I like it a lot!

Comment (darrynten) [reply src/Models/Company.php:574]: @make-it-git this functionality is great, we can get this in if we can add some information about this to the README.md file.

Does the `->get()` call also support this functionality?

Very nice

**Review by darrynten — CHANGES_REQUESTED** [review]

**Comment by make-it-git** [comment]
@darrynten updated

**Review by darrynten — APPROVED** [review]

**Comment by darrynten** [comment]
@make-it-git most excellent! Thanks

**Pull request merged**